package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "escooters",
+    "private": true,
+    "scripts": {
+        "dev": "vite",
+        "build": "vite build",
+        "lint": "./node_modules/.bin/eslint resources/js --ext .js,.vue",
+        "lintf": "./node_modules/.bin/eslint resources/js --ext .js,.vue --fix"
+    },
+    "dependencies": {
+        "@inertiajs/vue3": "^1.0.6",
+        "@tailwindcss/forms": "^0.5.3",
+        "@tailwindcss/typography": "^0.5.9",
+        "axios": "^1.4.0",
+        "laravel-vite-plugin": "^0.7.4",
+        "lodash": "^4.17.21",
+        "maska": "^2.1.9",
+        "tailwindcss": "^3.3.2",
+        "vue": "^3.2.47",
+        "vue-gallery": "^2.0.4",
+        "vue-toastification": "^2.0.0-rc.5"
```

Comment:
I think these packages are not needed right now:
- maska
- vue-gallery
- vue-toastification

Reply:
removed